content/examples/basic/front-matter/en.md

```diff
@@ -1,6 +1,13 @@
 ---
 title: Front Matter
 weight: 14
+date: "2023-08-01"
+meta: 
+- type: readingTime
+  value: 1
+- type: link
```

Comment:
discussed with Peter: we don't want to have a back link here because there already is a back link before the title so it's not needed and with all the other header items (authors, tags, etc) it will be to crowded

Reply:
This can be a link to anything though... still want to remove it?

Reply:
Yes I think so because it isn't in the designs.